platform/kobo/enable-wifi.sh

```diff
@@ -3,6 +3,8 @@
 # Load wifi modules and enable wifi.
 
 lsmod | grep -q sdio_wifi_pwr || insmod "/drivers/${PLATFORM}/wifi/sdio_wifi_pwr.ko"
+# Moar sleep!
+usleep 250000
```

Comment:
le sigh

Reply:
Not quite sure this one changes anything, but since we sleep after rmmod, I figured, why noit sleep after insmod, too :D.